marketplace/deployer_util/config_helper.py

```diff
@@ -669,10 +669,19 @@ def application_create(self):
 class SchemaXImage:
   """Accesses IMAGE and DEPLOYER_IMAGE properties."""
 
-  def __init__(self, dictionary):
+  def __init__(self, dictionary, default):
     self._split_by_colon = None
     self._split_to_registry_repo_tag = None
 
+    if not default:
+      raise InvalidSchema('default image value must be specified')
+    if not default.startswith('gcr.io'):
```

Comment:
Should we be concerned about marketplace.gcr.io or region based endpoints (ex: us-west1.gcr.io)?

Reply:
I advise against this particular check for gcr.io, since I don't think we need it. It does conform with the data we have seen so far and it is easy to change, so I am fine either way.

Reply:
Not yet. Our system currently **requires** partner staging GCR repositories to be in `gcr.io` ([container image solution documentation](https://cloud.google.com/marketplace/docs/partners/container-setup#tag_and_push_images); Kubernetes App documentation should have the same notice added since it has the same requirement)

We can update this check if we move to supporting other regions for staging repos.